tools/dissector/packet-opendds.cpp

```diff
@@ -451,7 +453,7 @@ namespace OpenDDS
 
                   // Get Entry Size
                   size = 0;
-                  gen_find_size(filter, size, padding);
+                  serialized_size(encoding, size, filter);
```

Comment:
```suggestion
                  size = serialized_size(encoding, filter);
```

Reply:
If I don't reset size back to 0 will it not cause problems?

Reply:
This is a template wrapper around the generated `serialized_size` that passes a zero to it and returns the result. It's equivalent, just a bit smaller.